### PR TITLE
fixed warning for OrgUpdate

### LINKED ISSUE #574: Rectify Errors/Warnings : `src/components/OrgUpdate/OrgUpdate.test.tsx`
**Describe the bug**
There are certain warning in the console when we run the test ```OrgUpdate.test.tsx``` using the command ```yarn test OrgUpdate.test```

**To Reproduce**

1. Open a terminal inside the Talawa Admin Project
2. Run the command ```yarn test OrgUpdate.test```
3. The tests will pass but, there would be many warning in the terminal.

#### Issue tracker for this file:
Make sure to complete the following issues.

* [ ] Removed the warning with statement ```This typically indicates a configuration error in your mocks setup, usually due to a typo or mismatched variable.```
* [ ] The terminal has no other warning

### PR Acceptance Criteria
- When complete this file must show 100% coverage when merged into the code base.
- [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin?search=&displayType=list)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide

Parent issue: #555 

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by akankshat05**
**What kind of change does this PR introduce?**

bugfix
**Issue Number:**

Fixes #574 

**Did you add tests for your changes?**

Yes
**Snapshots/Videos:**
orgupdate

**If relevant, did you update the documentation?**
No

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: src/components/OrgUpdate/OrgUpdate.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ function OrgUpdate(props: OrgUpdateProps): JSX.Element {
     try {
       const { data } = await login({
         variables: {
-          id: currentUrl,
+          id: props.id,
```
Comment (aviraldevv) [inline src/components/OrgUpdate/OrgUpdate.tsx:48]: @akankshat05 I didn't get why you've to change this it will break the functionality ;)

**Comment by anshgoyalevil** [comment]
@akankshat05 

Why have you changed the ```OrgUpdate.tsx``` file? It will break the functionality for Organization Update.

Please make changes to test file, and only make changes to Component file if necessary, keeping its functionality in mind.

**Review by anshgoyalevil — CHANGES_REQUESTED** [review]
Please replace ```props.id``` with ```currentUrl``` or you may implement a better and working approach for this file.

**Comment by akankshat05** [comment]
I will make the required changes.

**Comment by noman2002** [comment]
@akankshat05 Any progress on this ?

**Comment by akankshat05** [comment]
> @akankshat05 Any progress on this ?

I could not come up with a good solution other than hiding the warning. Should I make those changes?

**Comment by anshgoyalevil** [comment]
@akankshat05 

Yes, you may hide the warning using Mocklink for now.

Please resolve the changes requested above.

File: src/components/OrgUpdate/OrgUpdate.test.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ const MOCKS = [
     },
   },
 ];
-
+const mocklink = new MockLink(MOCKS, false, { showWarnings: false });
```
Comment (palisadoes) [inline src/components/OrgUpdate/OrgUpdate.test.tsx:105]: Hiding warnings is not a solution. Please remove this

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Comment by palisadoes** [comment]
Please fix the failing tests and the conflicting files. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained.

**Comment by akankshat05** [comment]
@palisadoes The issue which this PR was addressing was fixed by #745

**Comment by palisadoes** [comment]
Closing

**Pull request closed without merging**